src/java/picard/analysis/MeanQualityByCycle.java

```diff
@@ -56,7 +56,7 @@
         usageShort = MeanQualityByCycle.USAGE_SUMMARY,
         programGroup = Metrics.class
 )
-public class MeanQualityByCycle extends SinglePassSamProgram {
+public final class MeanQualityByCycle extends SinglePassSamProgram {
```

Comment:
It could probably be made final anyway.  No good reason to extend it as far as I can tell.